aiosmtpd/smtp.py

```diff
@@ -59,11 +59,13 @@ class _DataState(enum.Enum):
     "AuthMechanismType",
     "MISSING",
 ]  # Will be added to by @public
-__version__ = '1.2.3a4'
+__version__ = '1.3.0a1'
 __ident__ = 'Python SMTP {}'.format(__version__)
 log = logging.getLogger('mail.log')
 
 
+BOGUS_LIMIT = 5
```

Comment:
Yes, "bogus" commands are hard-limited to max of 5 per session.

I don't see why we need to make this customizable.